teacher.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Teacher < Person
+  def initialize(specialization)
```

Comment:
- [ ] Its a requirement that the constructor extends parent's constructor by adding @Specialization and a parameter for it
e.g
![image](https://user-images.githubusercontent.com/81932478/192602846-791e9bc4-3f39-469a-b0bf-3c862839849f.png)

Reply:
Thanks a lot @Munsa1  for pointing out these errors and taking out time to review the project. I will make the changes immediately.